Source/Krypton Components/Krypton.Toolkit/Utilities/General.cs

```diff
@@ -29,4 +31,43 @@ public enum KryptonColorButtonCustomColorPreviewShape
         RoundedSquare = 2,
         None = 3
     }
+
+    #endregion
+
+    #region Class: AssemblyUtilities
+
+    public class AssemblyUtilities
+    {
+        //https://www.meziantou.net/getting-the-date-of-build-of-a-dotnet-assembly-at-runtime.htm
+        public static DateTime GetLinkerTimestampUTC(Assembly assembly)
+        {
+            var location = assembly.Location;
+
+            return GetLinkerTimestampUTC(location);
+        }
+
+        public static DateTime GetLinkerTimestampUTC(string filePath)
+        {
+            const int PE_HEADER_OFFSET = 60;
+
+            const int LINKER_TIMESTAMP_OFFSET = 8;
+
+            var byteBuffer = new byte[2048];
+
+            using (var file = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                file.Read(byteBuffer, 0, byteBuffer.Length);
+            }
+
+            var headerPosition = BitConverter.ToInt32(byteBuffer, PE_HEADER_OFFSET);
+
+            var secondsSinceUNIXTimeStart = BitConverter.ToInt32(byteBuffer, headerPosition + LINKER_TIMESTAMP_OFFSET);
```

Comment:
What is going to happen ion 2034 ?

Reply:
> What is going to happen ion 2034 ?

Need to find a better solution, as it defaults to 1/1/1970 at the moment